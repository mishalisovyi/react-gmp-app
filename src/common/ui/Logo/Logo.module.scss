@use '../../../colors';
@use '../../../mixins';

.Logo {
	font-size: 18px;
	color: colors.$festive-fennec;
}

.Logo__bold-part {
	@include mixins.font-montserrat-black;
}

.Logo__regular-part {
	@include mixins.font-montserrat-medium;
}