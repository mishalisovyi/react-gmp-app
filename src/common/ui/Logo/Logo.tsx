import React, { memo } from 'react';
import styles from './Logo.module.scss';

function LogoComponent() {
  return (
    <div className={styles['Logo']}>
      <span className={styles['Logo__bold-part']}>netflix</span>
      <span className={styles['Logo__regular-part']}>roulette</span>
    </div>
  );
}

export const Logo = memo(LogoComponent);
