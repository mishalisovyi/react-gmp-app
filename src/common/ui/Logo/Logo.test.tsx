import React from 'react';
import renderer from 'react-test-renderer';

import { Logo } from './Logo';

it('should render logo', () => {
  const tree = renderer.create(<Logo />);

  expect(tree.toJSON()).toMatchSnapshot();
});
