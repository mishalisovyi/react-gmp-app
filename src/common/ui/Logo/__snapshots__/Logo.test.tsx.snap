// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render logo 1`] = `
<div>
  <span>
    netflix
  </span>
  <span>
    roulette
  </span>
</div>
`;
